### PR TITLE
Update feedback text strings

### LINKED ISSUE #524: Improve feedback strings of new assessments
We should improve the feedback strings of the new assessments to make them less wordy. 

**Known errors:**
subheadingDistributionTooLongAssessment: "The amount of words following after each of your subheadings **don't** exceed the recommended maximum of %1$d words, which is great."

subheadingDistributionTooShortAssessment: "The amount of words following after each of your subheadings **all** exceed the recommended minimum of %1$d words, which is great."

sentenceVariationAssessment: "The **sentence variation** score is 12.22, which is above the recommended minimum of 3. The text contains a nice combination of long and short sentences."

wordComplexityAssessment: ""%1$s of the words contain **over** %2$d syllables, which is less than or equal to the recommended maximum of %3$s.""

### PR DESCRIPTION
**Pull request by IreneStr**
Updates feedback strings for the following assessments:
- subheading length
- passive voice
- sentence length in text
- sentence variation
- subheading distribution too long and too short
- text length

Fixes #524

### CODE REVIEW

File: js/assessments/passiveVoiceAssessment.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ var calculatePassiveVoiceResult = function( passiveVoice, i18n ) {
 						"js-text-analysis",
 
 						// Translators: %1$d expands to the number of sentences in passive voice.
```
Comment (terw-dan) [inline js/assessments/passiveVoiceAssessment.js]: Can you rename Translators to translators? (without the capital T)?

File: js/assessments/passiveVoiceAssessment.js [hunk 2]
```diff
@@ -42,7 +42,7 @@ var calculatePassiveVoiceResult = function( passiveVoice, i18n ) {
 				"js-text-analysis",
 
 				// Translators: %1$d expands to the number of sentences in passive voice, %2$d expands to the recommended value
```
Comment (terw-dan) [inline js/assessments/passiveVoiceAssessment.js]: See comment above

File: js/assessments/sentenceLengthInTextAssessment.js [hunk 3]
```diff
@@ -53,7 +53,7 @@ var calculateSentenceLengthResult = function( sentences, i18n ) {
 			// Translators: %1$s expands to the percentage of sentences, %2$d expands to the maximum percentage of sentences.
```
Comment (terw-dan) [inline js/assessments/sentenceLengthInTextAssessment.js]: See comment above

**Comment by terw-dan** [comment]
CR Done. 
Could you please update the specs with the updated textstrings, these fail at the moment.

**Comment by atimmer** [comment]
CR & Acceptance done 👍🏻

**Pull request merged**